reconcile/slack_usergroups.py

```diff
@@ -70,12 +71,23 @@
         channels
         description
         ownersFromRepos
+        schedule {
+          schedule {
+            start
+            end
+            users {
+              org_username
+              slack_username
+            }
+          }
+        }
       }
     }
   }
 }
 """
 
+DATE_FORMAT = '%Y-%m-%d %H:%M'
```

Comment:
We have a lot of `DATE_FORMAT` constants across this repository. Can we either centralize them or agree on using `isoformat` everywhere?